hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -690,7 +690,7 @@ private static HoodieTableMetaClient newMetaClient(Configuration conf, String ba
         ? (HoodieTableMetaClient) ReflectionUtils.loadClass("org.apache.hudi.common.table.HoodieTableMetaserverClient",
         new Class<?>[]{Configuration.class, String.class, ConsistencyGuardConfig.class, String.class, FileSystemRetryConfig.class, String.class, String.class, HoodieMetaserverConfig.class},
         conf, basePath, consistencyGuardConfig, recordMergerStrategy, fileSystemRetryConfig,
-        metaserverConfig.getDatabaseName(), metaserverConfig.getTableName(), metaserverConfig)
+        Option.of(metaserverConfig.getDatabaseName()), Option.of(metaserverConfig.getTableName()), metaserverConfig)
         : new HoodieTableMetaClient(conf, basePath,
```

Comment:
How could the option be empty? Maybe you should use `Option.ofNullable`

Reply:
The table name can never be empty, but the database name may be empty.

is it necessary to change them all to `Option.ofNullable`?